students/meet_shah/lesson02/print_grid.py

```diff
@@ -0,0 +1,54 @@
+def print_grid(x):
+    topstart = "+ "
+    topmid   = "- "
+    topend   = "+"
+
+    midstart = "| "
+    midmid   = "  "
+    midend   = "|"
+
+    repeat = int(x/2)
+    topline = topstart + topmid*repeat + topstart + topmid*repeat + topend
+    midline = midstart + midmid*repeat + midstart + midmid*repeat + midend
+    
+    print(topline)
+    for i in range(0,2):
+        for i in range(0,repeat):
+            print(midline)
+        print(topline)
+ 
+def help_print_line(startpattern, endpattern, repeat):
+    for i in range(0,repeat):
```

Comment:
I like that you used a helper function to break up the code!
You can simplify this a bit by getting rid of the loop here and just using string multiplication. 
``` 
ret = startpattern * repeat + endpattern
print(ret)
```